services/QuillLMS/app/models/unit.rb

```diff
@@ -64,10 +60,12 @@ def hide_if_no_visible_unit_activities
   end
 
   def hide_classroom_units_and_unit_activities
```

Comment:
Does it make sense to change this method name now that we're not just doing the hide action here?

Reply:
Yes.  Good call